### DISCUSSION TITLE
Build and deploy your own copilot with GitHub Copilot 🚀 [Day 1 Universe]

### DISCUSSION THREAD

**Discussion by stebje** [community/community#74470] [Copilot Conversations]
## Build and deploy your own copilot with GitHub Copilot

**Speaker:** @vfm2 (Vanessa Minik, Service Delivery Engineer), @stebje (Steffen Bjerkenås, Staff Service Delivery Engineer)
**Time:** Wednesday, Nov 8, 12:30 PM - 2:00 PM PST

> In this session, we will dive into how you can build and deploy your own AI assistant, and how GitHub Copilot can help you throughout this process. First, we will cover some theory on GitHub Copilot functionalities and AI chatbot architectures, then we'll move onto a practical exercise where we build our own AI chatbot and deploy it to Azure—all while leveraging various features being explored through GitHub Copilot.

Thank you for joining our workshop! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and we'll do my best to answer. 

To watch the recording, please click [`link TBD`](https://github.com/orgs/community/discussions/74470).👈

**Comment by Destorntw** [comment]
Waiting to see the guide/materials..
If you have any free time I'd love to pick your brain on a model of deployment to a local hosted copilot on some GIGANTIC GPU's
Hi me up here 
Much love for the work on this! X P)

**Reply by mishmanners** [reply]
@vfm2 @stebje may be available to help you once they finish up IRL 👍

**Reply by stebje** [reply]
Hi there @Destorntw ! 👋 

The slides have been added to the [workshop repo](https://github.com/githubuniverseworkshops/workshop-build-deploy-your-own-copilot?tab=readme-ov-file#books-resources) 👍

**Comment by Xggq343w** [comment]
> ## Build and deploy your own copilot with GitHub Copilot
> **Speaker:** @vfm2 (Vanessa Minik, Service Delivery Engineer), @stebje (Steffen Bjerkenås, Staff Service Delivery Engineer) **Time:** Wednesday, Nov 8, 12:30 PM - 2:00 PM PST
> 
> > In this session, we will dive into how you can build and deploy your own AI assistant, and how GitHub Copilot can help you throughout this process. First, we will cover some theory on GitHub Copilot functionalities and AI chatbot architectures, then we'll move onto a practical exercise where we build our own AI chatbot and deploy it to Azure—all while leveraging various features being explored through GitHub Copilot.
> 
> Thank you for joining our workshop! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and we'll do my best to answer.
> 
> To watch the recording, please click [`link TBD`](https://github.com/orgs/community/discussions/74470).👈
